src/functional/ffront/type_info.py

```diff
@@ -97,27 +97,10 @@ def extract_dtype(symbol_type: ct.SymbolType) -> ct.ScalarType:
     raise GTTypeError(f"Can not unambiguosly extract data type from {symbol_type}!")
 
 
-def is_arithmetic(symbol_type: ct.SymbolType) -> bool:
-    """
-    Check if ``symbol_type`` is compatible with arithmetic operations.
-
-    Examples:
-    ---------
-    >>> is_arithmetic(ct.ScalarType(kind=ct.ScalarKind.FLOAT64))
-    True
-    >>> is_arithmetic(ct.ScalarType(kind=ct.ScalarKind.BOOL))
-    False
-    >>> is_arithmetic(ct.FieldType(dims=[], dtype=ct.ScalarType(kind=ct.ScalarKind.INT32)))
-    True
-    """
-    if extract_dtype(symbol_type).kind in [
-        ct.ScalarKind.INT32,
```

Comment:
There was a bug here: ct.ScalarKind.INT was missing. Now that I look at my PR, I would just revert and implement is_arithmetic as
`return is_floating_point(symbol_type) or is_integral(symbol_type)` 
let me know what you prefer.

Reply:
ok, maybe not a bug, but at least a inconsistency...

Reply:
`return is_floating_point(symbol_type) or is_integral(symbol_type)` looks good.